core/api/transport/impl/http_session.cpp

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/transport/impl/http_session.hpp"
+
+#include <boost/config.hpp>
+#include <outcome/outcome.hpp>
+
+namespace kagome::api {
+
+  HttpSession::HttpSession(Socket socket, Configuration config, Logger logger)
+      : config_{config},
+        stream_(std::move(socket)),
+        logger_{std::move(logger)} {}
+
+  void HttpSession::start() {
+    acyncRead();
+  }
+
+  void HttpSession::stop() {
+    boost::system::error_code ec;
+    stream_.socket().shutdown(Socket::shutdown_both, ec);
+    boost::ignore_unused(ec);
+  }
+
+  auto HttpSession::makeBadRequest(std::string_view message,
+                                   unsigned version,
+                                   bool keep_alive) {
+    Response<StringBody> res{boost::beast::http::status::bad_request, version};
+    res.set(boost::beast::http::field::server, kServerName);
+    res.set(boost::beast::http::field::content_type, "text/html");
```

Comment:
add `using namespace boost::beast::http`

Reply:
You probably looked changes from old commits. Now it is like `HttpField::server`, `HttpField::content_type`. I thinks it Is better than using whole namespace.